linux/devdoc/socket_transport_linux_requirements.md

```diff
@@ -344,4 +345,34 @@ MOCKABLE_FUNCTION(, bool, socket_transport_is_valid_socket, SOCKET_TRANSPORT_HAN
 
 **SOCKET_TRANSPORT_LINUX_11_094: [** If the socket inside `socket_transport_handle` is an `INVALID_SOCKET`, `socket_transport_is_valid_socket` shall fail and return `false`. **]**
 
-**SOCKET_TRANSPORT_LINUX_11_095: [** On success, `socket_transport_is_valid_socket` shall return `true`. **]**
\ No newline at end of file
+**SOCKET_TRANSPORT_LINUX_11_095: [** On success, `socket_transport_is_valid_socket` shall return `true`. **]**
+
+### socket_transport_get_local_address
+
+```c
+MOCKABLE_FUNCTION(, int, socket_transport_get_local_address, SOCKET_TRANSPORT_HANDLE, socket_transport, char, hostname[MAX_GET_HOST_NAME_LEN], LOCAL_ADDRESS**, local_address_list, uint32_t*, address_count);
```

Comment:
>socket_transport_get_local_address [](http://example.com/codeflow?start=25&length=34)

Is there a way we can specify the interface card or default if that field is NULL?

Reply:
I thought about that, but the problem is we don't know what interface card to specify.  We don't have the name of them or the value.

Reply:
Okay, usually developers just provide the string field and run some command to get all available NIC cards and compare them to the input value. But not needed for this PR, I guess.